### PR TITLE
docs: add angular with vite example

### PR DESCRIPTION
**Pull request by drdreo**
I've created a plain Angular project using their CLI. With ng version 17.3 it also comes with vite so it could fit into that category.
The project also demos quickly how to use the plugins to get started.

### Intention
I was not finding a single reference regarding Angular in the UnoCSS documentation which lead to the conclusion Angular is not supported. This is very much the opposite and I want to help to change that perception. 

It would have great benefits having at least a demo example referenced inside the official docs. This could spread awareness of this great tool amongst the ng-folk.

### CODE REVIEW

**Comment by zyyv** [comment]
That's a great start, and you can guide the other example, remove some unnecessary files, make it minimal.

**Comment by drdreo** [comment]
@zyyv thanks for the feedback. I have no removed all obsolete files, to have a minimal, servable project and also added the common stackblitz file (assuming this is needed for demo purposes).
I also added the entry to the examples cards here: [https://unocss.dev/integrations/#examples](https://unocss.dev/integrations/#examples)

Anything else you want me to add?

File: examples/vite-angular/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "@types/jasmine": "~5.1.0",
+    "jasmine-core": "~5.1.0",
+    "karma": "~6.4.0",
+    "karma-chrome-launcher": "~3.2.0",
+    "karma-coverage": "~2.2.0",
+    "karma-jasmine": "~5.1.0",
+    "karma-jasmine-html-reporter": "~2.1.0",
+    "unocss": "^0.60.0",
+    "typescript": "~5.4.2"
+  }
```
Comment (userquin) [inline examples/vite-angular/package.json:40]: Add this entry:
```json
  "stackblitz": {
    "installDependencies": false,
    "startCommand": "node .stackblitz.mjs && npm install && npm run dev"
  }
```

File: examples/vite-angular/.stackblitz.mjs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import fs from 'node:fs'
+
+updateDependencyLinksToLatest('./package.json')
+
+function updateDependencyLinksToLatest(filename) {
+  try {
+    const contents = fs.readFileSync(filename, 'utf-8')
+    const updatedContent = contents.replace(/"link:...+"/gi, '"latest"')
```
Comment (userquin) [inline examples/vite-angular/.stackblitz.mjs:8]: can you change the regex to: `/"link:.{3,}"/gi`?

Comment (userquin) [reply examples/vite-angular/.stackblitz.mjs:8]: for context: https://github.com/unocss/unocss/pull/3801

File: examples/vite-angular/package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "ng-uno",
+  "version": "0.0.0",
```
Comment (userquin) [inline examples/vite-angular/package.json:3]: shouldn't add `"type": "module"`?

Comment (userquin) [reply examples/vite-angular/package.json:3]: UnoCSS is ESM only...

**Comment by userquin** [comment]
> I also added the entry to the examples cards here: https://unocss.dev/integrations/#examples But the `i-logos-angular` doesn't seem to exist yet.

https://icones.js.org/collection/logos?s=angular

**Comment by userquin** [comment]
@drdreo you can test your branch in SB: https://stackblitz.com/github/drdreo/unocss/tree/docs-vite-angular/examples/vite-angular

**Comment by userquin** [comment]
@drdreo can you run in root folder `nr lint` to fix lint errors?

**Comment by drdreo** [comment]
@userquin My apologies, i had to setup pnpm first. Now its linted properly and i factored in your feedback.

**Comment by userquin** [comment]
I'll check the PR later...

**Comment by drdreo** [comment]
Yeah, so before Angular builds, we need the referenced `uno.css` file which is generated by the unocss cli.
For that i have added the `"dev": "npm run build:css && npm run start",` script which works for me locally when ran. This is also used in the stackblitz.mjs but it seems the preview didn't update yet.

**Comment by userquin** [comment]
I'm going to include uno.css in `.gitignores` and fix pnpm workspace for the example... I'm also including `cli.analytics = false`  in angular.json.

**Comment by userquin** [comment]
No idea what's wrong with my previous VP link, cloned repo can be found here: https://stackblitz.com/edit/stackblitz-starters-gybg14?file=package.json,src%2Funo.css

**Review by userquin — APPROVED** [review]

**Pull request merged**